### PR TITLE
Fixes based on @hugo-ter-doest's comments on #3

### PR DESCRIPTION
**Pull request by ElenaFdR**
I've created this as a draft PR while I work through @hugo-ter-doest's comments.

Still to do:
~~- [ ] standardize jullie vs uw throughout the document~~ (logged as #8)
- [x] substantive changes in governance doc (in both Dutch and English)

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -80,9 +80,9 @@ When a consensus cannot be found a team member can call for a majority vote on a
 Many of the day-to-day project maintenance tasks can be done by a lazy consensus model. But the following items must be called to vote:
 
 * Adding a team member (simple majority)
-* Removing a team member (super majority)
-* Changing the governance rules (this document) (super majority) 
-* Licensing and intellectual property changes (including new logos, wordmarks) (simple majority)
+* Removing a team member (supermajority)
+* Changing the governance rules (this document) (supermajority)
+* Licensing and intellectual property changes (including new logos, wordmarks) (supermajority)
```
Comment (clausmullie) [inline GOVERNANCE.md:85]: @ElenaFdR is this change from simple to supermajority intentional?

Comment (Hugo-ter-Doest) [reply GOVERNANCE.md:85]: From my part it is. Changing licensing and IPR may have a big impact on OpenZaak. Therefore, a simple majority is not enough.

**Review by Ainali — APPROVED** [review]
I only reviewed the changes in English, but those looked all good to me

**Comment by clausmullie** [comment]
Checked the Dutch parts, looks good!

**Pull request merged**